util/testutil/workers/oci_windows.go

```diff
@@ -0,0 +1,7 @@
+package workers
+
+//lint:ignore U1000 for parity with unix
+func InitOCIWorker() {
```

Comment:
since we'd a couple of instances where `InitOCIWorker()` was being called, seemed cleaner to go with the no-op solution as opposed to having `if runtime.GOOS == "windows"` on each of those instances:

```
$ grep -rn "workers.InitOCIWorker()"
client/client_test.go:80:               workers.InitOCIWorker()
cmd/buildctl/buildctl_test.go:15:       workers.InitOCIWorker()
frontend/dockerfile/dockerfile_test.go:62:              workers.InitOCIWorker()
frontend/frontend_test.go:25:           workers.InitOCIWorker()
solver/jobs_test.go:16: workers.InitOCIWorker()
```

Reply:
unless there are some platform specific imports that fail on Windows, I think we can have just one copy of the `InitOCIWorker()` and do a:

```go
if runtime.GOOS == "windows" {
    return
}
```
at the beginning of the function, with a log message stating that it's not supported on Windows. We don't want it to silently exit if the function is actually called. Users of the function should be aware that the OCI worker is not supported on Windows.

The function should also be commented stating clearly that the OCI worker is not supported on Windows. Something like:

```md
// InitOCIWorker registers an integration test worker, which enables the --oci-worker
// flag in the test buildkitd instance and disables the --containerd-worker flag. This
// integration test worker is not supported on Windows. 
```

It makes sense to have separate definitions of the function based on platform if there are platform specific implementations of it. But I have no strong preference here.

Reply:
Fixing the comments.

Do you think there will any efforts in the future to bring this to Windows? If so, splitting out might be a better approach, so we won't need to go reverting in all the `if` instances. 

Also any new instance of `workers.InitOCIWorker` won't need to be guarded every time, most likely for folks building for other platforms and may not need to care that this isn't support on Windows yet.

Reply:
Not sure the OCI worker will be supported on Windows.

You can split them in platform specific files, but I would suggest you create new, unexported functions (like `initOCIWorker()`) for each platform, that you call from the exported `InitOCIWorker()` function.

The comment I mentioned should be added to the exported function and the windows version should still log that it is not implement on Windows.

This way, anyone that stumbles across this code in the future will have an easier time understanding it.

Reply:
Ah, got it, makes sense 👍